src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -106,8 +122,8 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
                         model.expectedCount = Integer.valueOf(recordDetails.get('Expected_Count_of_Gifts__c'));
                         model.expectedTotal = (Decimal)recordDetails.get('Expected_Total_Batch_Amount__c');
                     }
-                } catch (Exception ex) {
```

Comment:
I don't mind the change but I am curious about why you prefer one over the other ...